keyboards/handwired/d48/glcdfont_d48.c

```diff
@@ -0,0 +1,240 @@
+#pragma once
+
+#ifdef __AVR__
+ #include <avr/io.h>
+ #include <avr/pgmspace.h>
+#elif defined(ESP8266)
+ #include <pgmspace.h>
+#else
+ #define PROGMEM
+#endif
```

Comment:
I think you only need: 
```suggestion
#include "progmem.h"
```